### PR TITLE
Strata/Platform - UI: Certify checkboxes

### PR DESCRIPTION
**Pull request by deetz99**
*Issue:*

- bcgov/entity/issues/24515
- bcgov/entity/issues/24516

*Description of changes:*
- new common component for certify checkbox list (accept text or i18n key with dynamic props)
- update strata and platforms confirmation checkboxes
- 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

File: strr-host-pm-web/app/components/form/UnitAddress/AutoComplete.vue
Comment (deetz99) [inline strr-host-pm-web/app/components/form/UnitAddress/AutoComplete.vue:1]: this was duplicating the city province and postal code in the address display

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://strr-platform-dev--pr-356-v1bjobsd.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://strr-strata-hotel-dev--pr-356-b3w6oa9s.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://strr-strata-hotel-dev--pr-356-b3w6oa9s.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://strr-platform-dev--pr-356-v1bjobsd.web.app

File: strr-base-web/app/components/connect/form/certify/index.vue [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+        <UCheckbox
+          v-model="checkboxModel"
+          class="pt-4"
+          aria-required="true"
+          :aria-describedby="items.map((_, i) => `${name}-item-${i}`).join(' ')"
+          :aria-invalid="hasError"
+        >
+          <template #label>
+            <slot name="checkboxLabel">
+              <ConnectI18nBold
```
Comment (deetz99) [inline strr-base-web/app/components/connect/form/certify/index.vue:61]: For checkbox with the users name, should be able to pass something like 

```
checkboxLabel: { key: 'key.to.checkbox.label', props: { name: username } }
```

Can do something similar with the item i18nkey and i18nprops

File: strr-strata-web/app/components/form/ReviewConfirm.vue [hunk 2]
```diff
@@ -312,43 +312,37 @@ onMounted(async () => {
...
+      ref="strataConfirmationFormRef"
+      :state="applicationStore.confirmation"
+      :schema="applicationStore.confirmationSchema"
+      class="space-y-10 pb-10"
+    >
+      <ConnectFormCertify
+        v-model="applicationStore.confirmation.confirmation"
+        :title="$t('strr.label.confirmation')"
+        :items="[{ slot: 'item-1' }, { i18nKey: 'certify.2' }]"
+        :checkbox-label="{ key: 'certify.checkbox' }"
```
Comment (deetz99) [inline strr-strata-web/app/components/form/ReviewConfirm.vue:325]: can pass this a regular string or an i18n key with whatever props you want eg: name

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://strr-platform-dev--pr-356-v1bjobsd.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://strr-strata-hotel-dev--pr-356-b3w6oa9s.web.app

**Review by thorwolpert — APPROVED** [review]

**Review by kialj876 — APPROVED** [review]
Nice work

**Pull request merged**